Project/Homebrew/README.md

```diff
@@ -0,0 +1,22 @@
+Building
+--------
+
+Install ffmpeg, freetype2, qt5
+
+    $ brew install ffmpeg freetype2 qt5
+
```

Comment:
[qt5 is keg_only](https://github.com/Homebrew/homebrew/blob/master/Library/Formula/qt5.rb#L64) so is not linked by default. Some doc on this is here; https://github.com/Homebrew/homebrew/blob/15f291ddaa05cc010456a733b799382175c29dd4/share/doc/homebrew/How-to-build-software-outside-Homebrew-with-Homebrew-keg-only-dependencies.md#how-do-i-get-non-homebrew-builds-to-find-those-tools.